### DISCUSSION TITLE
simple-ortho bug

### DISCUSSION THREAD

**Discussion by aldurgham1** [community/community#55221] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, the simple-ortho code runs really nice, i love it.
I am facing one issue, the output image looks pixalized/bulky non-smooth. I tried all interpolation combinations and i tried to force constant resolution between input and output but was unable to fix.

Any idea how to fix that?

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/leftfield-geospatial/simple-ortho? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:
